### ISSUE TITLE
spec: get rid of fulfills for the volumes

### ISSUE THREAD

**Issue by philips** [appc/spec#13]
_From @philips on December 3, 2014 2:48_

Right now volumes in the container can fulfill particular labels. This is a bit troublesome because there might be colliding names for the same volume.

https://github.com/coreos/rocket/blob/master/app-container/SPEC.md#container-runtime-manifest-schema

Instead of doing this give each volume a name and each app will get volumes filled in by name to its mountpoints.

```
    "apps": [
        {
            "app": "example.com/reduce-worker-1.0.0",
            "imageID": "sha256-277205b3ae3eb3a8e042a62ae46934b470e431ac",
            "mountPoints": [{"src": "work", "dest": "data"}]
        },
        {
            "app": "example.com/worker-backup-1.0.0",
            "imageID": "sha256-3e86b59982e49066c5d813af1c2e2579cbf573de",
             "mountPoints": [{"src": "buildoutput", "dest": "backup"}]
        },
    ],
    "volumes": [
        {"name": "work", "kind": "host", "source": "/opt/tenant1/work", "readOnly": true},
        {"name": "buildoutput", "kind": "empty"}
    ],
```

_Copied from original issue: coreos/rocket#182_

**Comment by philips** [comment]
I am working on this now.

**Comment by jonboulle** [comment]
ping @philips

**Comment by philips** [comment]
@jonboulle sorry, I am going to get this up today :-/

**Comment by jonboulle** [comment]
Fixed by #143
